tests/driver_hcsr04/main.c

```diff
@@ -0,0 +1,116 @@
+#include <stdio.h>
+#include "xtimer.h"
+#include "assert.h"
+
+#include "phydat.h"
+#include "saul_reg.h"
+
+#include "hcsr04_params.h"
+#include "hcsr04.h"
+
+
+#define TEST_TIME                               (3e6)
+#define TEST_CONVERTION_MACRO(x)                
+
+
+int main(void) 
+{
+    puts("********** Test HC-SR04 RIOT driver! **********");
+
+    // deactivate saul auto-initialization, due to conflict when both work at the same time
```

Comment:
May as well use the `/* */`-style of comments (to follow the existing style in RIOT's codebase) both here and in the driver code :wink: